### ISSUE TITLE
Target cell

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7537]
**Preferred term label:**
Target cell

**Synonyms**
Codocyte

**Definition (free text, please give PubMed ID)**
Target cells (codocytes) have a centrally located disk of hemoglobin surrounded by an area of pallor with an outer rim of hemoglobin adjacent to the cell membrane giving the cell the appearance of a target.
PMID: 32965812  PMID: 27372866

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
